src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusManualProcessing.kt

```diff
@@ -41,30 +62,61 @@ fun handleStatusMANUALPROCESSING(
     kafkaproducerreceivedSykmelding: KafkaProducer<String, ReceivedSykmelding>,
     sm2013ManualHandlingTopic: String,
     kafkaproducervalidationResult: KafkaProducer<String, ValidationResult>,
-    sm2013BehandlingsUtfallToipic: String
+    sm2013BehandlingsUtfallToipic: String,
+    kafkaproducerManuellOppgave: KafkaProducer<String, ManuellOppgave>,
+    syfoSmManuellTopic: String,
+    personV3: PersonV3,
+    arbeidsfordelingV1: ArbeidsfordelingV1
 ) {
-    opprettOppgave(kafkaManuelTaskProducer, receivedSykmelding, validationResult, loggingMeta)
-
-    notifySyfoService(session = session, receiptProducer = syfoserviceProducer, ediLoggId = ediLoggId,
-            sykmeldingId = receivedSykmelding.sykmelding.id, msgId = msgId, healthInformation = healthInformation)
-    log.info("Message send to syfoService {}, {}", syfoserviceQueueName, StructuredArguments.fields(loggingMeta))
-
-    kafkaproducerreceivedSykmelding.send(ProducerRecord(sm2013ManualHandlingTopic, receivedSykmelding.sykmelding.id, receivedSykmelding))
-    log.info("Message send to kafka {}, {}", sm2013ManualHandlingTopic, StructuredArguments.fields(loggingMeta))
-
-    sendValidationResult(validationResult, kafkaproducervalidationResult, sm2013BehandlingsUtfallToipic, receivedSykmelding, loggingMeta)
-
-    val apprec = fellesformat.toApprec(
-            ediLoggId,
-            msgId,
-            msgHead,
-            ApprecStatus.OK,
-            null,
-            msgHead.msgInfo.receiver.organisation,
-            msgHead.msgInfo.sender.organisation
-    )
-    sendReceipt(apprec, sm2013ApprecTopic, kafkaproducerApprec)
-    log.info("Apprec receipt sent to kafka topic {}, {}", sm2013ApprecTopic, StructuredArguments.fields(loggingMeta))
+
+    val geografiskTilknytning = fetchGeografiskTilknytning(personV3, receivedSykmelding)
+    val patientDiskresjonsKode = fetchDiskresjonsKode(personV3, receivedSykmelding)
+    val finnBehandlendeEnhetListeResponse = fetchBehandlendeEnhet(arbeidsfordelingV1, geografiskTilknytning.geografiskTilknytning, patientDiskresjonsKode)
```

Comment:
Her skal vi vel egentlig også ha med hvorvidt det er snakk om en egen ansatt eller ikke? Tror dessverre det kun finnes en ws for sjekk av egen ansatt pt. :/

Reply:
Eg legger til egenansatt kall

Reply:
Vi bør legge det til, gleder meg til å komme ut av pilot, da kan vi droppe alt dette å la oppgave gjøre det